lib/bridge/IrcHandler.js

```diff
@@ -538,14 +550,16 @@ IrcHandler.prototype.onJoin = Promise.coroutine(function*(req, server, joiningUs
         const joinRetry = (attempts) => {
             req.log.debug(`Joining room (attempts:${attempts})`);
             return intent.join(room.getId()).catch((err) => {
-                if (attempts > MAX_JOIN_ATTEMPTS || err.httpStatus === undefined || err.httpStatus <= 500) {
+                if (attempts > MAX_JOIN_ATTEMPTS
+                    ) {
                     req.log.error(`Not retrying join for ${room.getId()}.`);
                     return Promise.reject(err);
                 }
                 attempts++;
                 const delay = JOIN_DELAY_MS*attempts;
-                req.log.warn(`Failed to join ${room.getId()} attempts:${attempts} ${err}.delaying for ${delay}ms`);
-                req.log.debug(`Failed with: ${err.errcode} ${err.message}`);
+                req.log.warn(
+                `Failed to join ${room.getId()} attempts:${attempts} Delaying for ${delay}ms`
```

Comment:
Is there a reason why this is on a different line (and if so, why isn't it tabbed in?)

Reply:
I suspect the linter was screaming at me. It should be tabbed, though.